app/controllers/sessions_controller.rb

```diff
@@ -3,18 +3,37 @@ def new
   end
 
   def create
-    if params[:email].present?
-      user = User.find_or_create_by!(email: params[:email])
+    if auth_email.present?
+      user = User.find_or_create_by!(email: auth_email)
```

Comment:
asi email by mal ist cez downcase. chceme ho unikatny aj ked niekto omylom zada velke pismeno (napr. na mobile)

Reply:
toto urobim, navrhujem ukladat to co uzivatel zadal ale porovnavat to v rovnakom case, napr. downcase

Reply:
V tom pripade ale treba function index v db, aby to hladalo rychlo.

Reply:
s tym pocitam